index.md

```diff
@@ -187,41 +187,36 @@ print(a[0:3])
 
 ### E1128: Assignment from None {#E1128}
 
-This error occurs when you assign a variable the return value of a function call,
-but the function always returns `None`.
-This error is similar to E1111.
-In the following example, `f` always returns `None`. As a result, `x` will always
-get the value `None`.
+This error occurs when you assign a variable the return value of a function call, but the function always returns `None`. This error is similar to [E1111](#E1111).
+
+In the following example, `add_fruit` always returns `None`.  As a result, `new_fruit_basket` will always get the value `None`.
 
 ~~~~ {include="E1128_assignment_from_none"}
 ~~~~
 
 
 ### E1130: Invalid unary operand type {#E1130}
 
-This error occurs when the unary operand on the objects does not support this
-type of operation. For example, we should never add string to integer.
+This error occurs when you use a [unary operator][2] (`+`, `-`, `~`) on an object which does not support this operator. For example, a list does not support negation.
 
 ~~~~ {include="E1130_invalid_unary_operand_type"}
 ~~~~
 
 
 ### E1131: Unsupported binary operation {#E1131}
 
-This error occurs when you use a binary arithmetic operator like `+` or `*`,
-but the left and right sides are not compatible types.
-For example, a dictionary cannot be added to a list.
+This error occurs when you use a [binary arithmetic operator][3] like `+` or `*`, but the left and right sides are not compatible types. For example, a dictionary cannot be added to a list.
 
 ~~~~ {include="E1131_unsupported_binary_operation"}
 ~~~~
 
 
 ### E1135: Unsupported membership test {#E1135}
 
-This error occurs when you use the membership test `a in b`, but the `b`'s type
-does not support this type of check.
+This error occurs when you use the membership test `a in b`, but the type of `b`
```

Comment:
Please avoid using the word "you", here and in other instances.  I believe this has been mentioned before.
There are many ways a phrase could be reworded, for example, 
> This error occurs when the membership of `a` is tested in `b`, i.e.  `a in b`, but the type of `b` does not support membership tests.